src/components/academies/AcademyDetailPage.jsx

```diff
@@ -54,45 +52,26 @@ const AcademyDetailPage = () => {
 
   return (
     <Container size="lg" className="pt-3 pb-4">
-      {isAcademyAPILoading
-        ? <Skeleton height={30} count={0.25} className="mb-4" data-testid="academy-breadcrumbs-loading" />
-        : (
-          <div className="small">
-            <Breadcrumb
-              data-testid="academy-breadcrumb"
-              links={[
-                { label: 'Find a Course', to: `/${enterpriseCustomer.slug}/search` },
-              ]}
-              linkAs={Link}
-              activeLabel={academy?.title}
-            />
-          </div>
-        )}
+      <div className="small">
+        <Breadcrumb
+          data-testid="academy-breadcrumb"
+          links={[
+            { label: 'Find a Course', to: `/${enterpriseCustomer.slug}/search` },
+          ]}
+          linkAs={Link}
+          activeLabel={academy?.title}
```

Comment:
nit: consider removing the `?.` when accessing properties on `academy` here and below, given the `!academy` check above.